### PR TITLE
Update website documentation

### PR DESCRIPTION
**Pull request by quantumm**
Introduce a Quick start page.
Try to clean the structure of the documentation.
Some dead pages have been removed.

Hope you like it ;)

### CODE REVIEW

File: docs/src/quick_start.rst [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+Numerous plugins are already available for common scientific equipment suppliers, they are referenced in the
+`list of supported instruments <https://github.com/PyMoDAQ/pymodaq_plugin_manager/blob/main/README.md>`_.
+
+.. figure:: /image/quick_start/supported_instruments_list.png
+
+   List of supported instruments.
+
+We can access it from the *Supported instruments* link on the website’s home page.
+
+.. figure:: /image/quick_start/supported_instruments.png
```
Comment (seb5g) [inline docs/src/quick_start.rst]: you could change this or just remove it as your statement above is clear enough.

File: docs/src/quick_start.rst [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+
+``plugin_manager``
+
+A window is displayed to easily install the plugin.
+
+.. figure:: /image/quick_start/plugin_manager.png
+   :width: 400
+
+   The Plugin Manager interface.
+
```
Comment (seb5g) [inline docs/src/quick_start.rst:147]: could you add a note here giving the pip install command for those who do not need the plugin manager?

File: docs/src/quick_start.rst [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+We then have to select the correct *DAQ type* (here 2D because the detector outputs a matrix of pixels) (1), the
+corresponding plugin (2), the specific camera through its serial number (3), initialize the communication with the
+camera (4), and get a snapshot (5).
+
+.. figure:: /image/quick_start/quick_start_image_thorlabs.png
+   :width: 800
+
+   Snapshot from a Thorlabs camera equipped with an objective, in a DAQ Viewer.
+
```
Comment (seb5g) [inline docs/src/quick_start.rst:167]: I like your snapshot!

File: docs/src/glossary.rst [hunk 4]
```diff
@@ -39,22 +37,35 @@ Here are some definitions of the specific terms used in the PyMoDAQ documentatio
...
+     python file containing data, functions or classes.
+
+.. note::
+    If there is code that can be executed within your module but you don't want it to be executed when importing,
+    make sure to protect the execution using a : ``if __name__ == '__main__':`` clause.
+
+.. glossary::
+
+  Navigation
+    See above.
```
Comment (seb5g) [inline docs/src/glossary.rst]: should be put just below the Signal term

**Review by seb5g — CHANGES_REQUESTED** [review]
Hi @quantumm thanks for this nice work. I raised some minor points in my review!

**Comment by seb5g** [comment]
I also noted that we can move to docutils==0.18.1 in the requirements.txt file in ./docs/ Could you change it in your PR?

File: docs/src/quick_start.rst (hunk 1 above)
Comment (quantumm) [reply docs/src/quick_start.rst]: Done.

File: docs/src/quick_start.rst (hunk 2 above)
Comment (quantumm) [reply docs/src/quick_start.rst:147]: Done.

File: docs/src/quick_start.rst (hunk 3 above)
Comment (quantumm) [reply docs/src/quick_start.rst:167]: It took me a bit of time to know what I would snapshot! :P

File: docs/src/glossary.rst (hunk 4 above)
Comment (quantumm) [reply docs/src/glossary.rst]: This change happened because i reordered alphabetically the terms.
I added a link to the Signal term.

**Comment by quantumm** [comment]
I made all the corrections.

Actually the version of docutils I had in my environment was something like 0.21.... I don’t know why.

But I tried the build with 0.18 it is working.

**Review by seb5g — APPROVED** [review]

**Pull request merged**